lib/src/facts/linux/virtualization_resolver.cc

```diff
@@ -170,12 +170,14 @@ namespace facter { namespace facts { namespace linux {
 
     string virtualization_resolver::get_vmware_vm()
     {
-        auto result = execute("vmware", { "-v" });
-        if (!result.first) {
+        bool success;
+        string output, error;
+        tie(success, output, error) = execute("vmware", { "-v" });
```

Comment:
Same, change error to something that makes it obvious it's a dummy because stderr redirects to null.